include/dynamatic/Support/CFG.h

```diff
@@ -97,8 +97,26 @@ bool isBackedge(Value val, Operation *user, BBEndpoints *endpoints = nullptr);
 /// user (the function will assert if that is not the case).
 bool isBackedge(Value val, BBEndpoints *endpoints = nullptr);
 
-/// Determines whether the Handshake operation cannot belong to the implicit CFG
-/// of a Handshake function (i.e., cannot have a basic block attribute).
+/// Data structure to hold all edges belonging to a pair of endpoints. This
+/// constitutes an Arc. srcBB and dstBB can be equal when the arcs are
+/// Backedges. An edge is represented by means of an mlir::OpOperand.
+struct BBArc {
+  unsigned srcBB;
```

Comment:
All class members should have a short comment e.g.,
```cpp
/// The edge's source basic block.
unsigned srcBB;
```

This is helpful when using the class to get a description of what each member is on the go, even it is a bit trivial like here.